lib/constants.cjs

```diff
@@ -9,16 +9,21 @@ const DEFAULT_IGNORE_FILENAME = '.stylelintignore';
 
 const DEFAULT_FORMATTER = 'string';
 
+// NOTE: Partially based on `sysexits.h`.
 const EXIT_CODE_SUCCESS = 0;
-const EXIT_CODE_FATAL = 1;
-const EXIT_CODE_ERROR = 2;
+const EXIT_CODE_FATAL_ERROR = 1;
+const EXIT_CODE_LINT_PROBLEM = 2;
+const EXIT_CODE_INVALID_USAGE = 64;
+const EXIT_CODE_INVALID_CONFIG = 78;
```

Comment:
Do we need a new import for https://github.com/stylelint/stylelint/blob/v16/lib/__tests__/extends.test.mjs#L75?

Reply:
Ah, sorry. What do you mean by "additional import" in the comment?

Reply:
> Ah, sorry. What do you mean by "additional import" in the comment?

```
import { EXIT_CODE_INVALID_CONFIG } from '../foo/bar/constants.mjs';
```

Reply:
I see. I've fixed the test also via 722f648fd. The test code now becomes more readable!